source/internal.d.ts

```diff
@@ -42,3 +42,12 @@ export type Subtract<A extends number, B extends number> = BuildTuple<A> extends
 Matches any primitive, `Date`, or `RegExp` value.
 */
 export type BuiltIns = Primitive | Date | RegExp;
+
+/**
+Returns the accessible keys that also works for union type.
```

Comment:
No idea what this is trying to say. But from reading the SO answer, I guess you meant something like this:

> Get all the keys from types in a union type.